sources/editor/Stride.Assets.Presentation/AssetEditors/EntityHierarchyEditor/Game/EditorGameEntityCameraService.cs

```diff
@@ -248,7 +261,7 @@ protected void UpdateCameraBase(ref float yaw, ref float pitch, ref Vector3 posi
             if (input.isPanning)
             {
                 float panningSpeed = asOrthographic ? Component.OrthographicSize : revolutionRadius;
-                panningSpeed *= MouseMoveSpeedFactor * baseSpeed;
+                panningSpeed *= (MouseMoveSpeedFactor * baseSpeed) / 30f;
```

Comment:
Any reason for this division here? Shouldn't baseSpeed be modified instead?

Reply:
The intention here was to reduce the calculated panning speed (depending on the current movement speed) at a constant fraction to have a better feeling. This value was evaluated during testing.

So panning speed should be higher if the movement speed is higher or lower if the movement speed gets lower, too. But not "equal", to not be too fast. Its adapting some fraction of the current choosen movement speed.

Reply:
Ok, since this is called every frame, can you make the division become multiplication instead and extract the constant 1/30 like
```csharp
const float panningSpeedModifier = 1/30f;
```

Reply:
I have implemented your mention. Its now a fixed value, which I have predefinde, too.

Additionally I have extended the increments a little further and improved the Camera speed settings in the UI.
It now shows the current speedvalue and the Slider is a little wider, to have better control over the extended increments.

Here you can see the changes to the UI :)
![Camera_Speedsettings](https://github.com/stride3d/stride/assets/44621949/84a46769-47bc-42a9-a3c9-077fbad4ddb7)

I'm done now with all I wanted to do. Hope it is okay now and can be merged :)